proposal.rst

```diff
@@ -313,4 +334,4 @@ you plan to use, please give details.**
 .. to clearly identify that code (and keep existing licensing and copyright
 .. details intact), and to check with the mentors that it is OK to use.
 
-FILLME
+I don't know exactly, maybe some third-party api for interacting with big datasets will be needed.
```

Comment:
What problems do you think you might encounter that would lead to needing this? Where would they fit within Xapian (which itself is already a system for dealing with at least medium-sized datasets, depending on your definition of size)?